app/controllers/books_controller.rb

```diff
@@ -0,0 +1,5 @@
+class BooksController <  ApplicationController
```

Comment:
Unnecessary spacing detected.<br>Operator `<` should be surrounded with a single space.